src/scripts.js

```diff
@@ -170,13 +169,15 @@ getData('hydration/hydrationData').then(function(hydrationData) {
       },
       body: JSON.stringify({
         userID: user.id,
-        date: `${dateInput}`,
+        date: date,
         numOunces: ounceInput
       })
     })
+    .then(response => response.json())
+    .then(data => $('#water-consumed').text(`${data.numOunces} Ounces \n\n`))
```

Comment:
Nice updating this! So does the .then on 177 then display the new ounces consumed that the user just put in? It looks like this is using the promise that results from the fetch-POST to display that data?

Reply:
Yes it does. It updates on the tile, but does not update on the chart. The chart populates using the method that gets a week's worth of data. I didn't chase that rabbit hole today of how to update only the most recent day (which equates to only one bar on the chart changing). The whole chart populates at the same time on one method, so changing the last day is a whole process that I didn't think would be wise to chase at this time.